src/components/CounterBubble/CounterBubble.vue

```diff
@@ -0,0 +1,82 @@
+<!--
+ - @copyright Copyright (c) 2019 Marco Ambrosini <marcoambrosini@pm.me>
+ -
+ - @author Marco Ambrosini <marcoambrosini@pm.me>
```

Comment:
@PVince81 Probably not really important, but I guess this is the wrong year and author?